contrib/hack/test_walkthrough.sh

```diff
@@ -135,6 +135,10 @@ retry -n 10 \
 
 echo 'Waiting on pods to come up...'
 
+wait_for_expected_output -x -e 'Pending' -n 10 \
```

Comment:
@kibbles-n-bytes I don't think I understand what the line this comment is attached to is doing then - can you clarify?